src/Compilers/VisualBasic/Portable/Symbols/Source/SourceMemberMethodSymbol.vb

```diff
@@ -697,17 +697,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
                 Return Nothing
             End If
 
-            typeBinder.ReportUseSiteError(diagBag, singleHandleClause.EventMember, eventContainingType)
```

Comment:
Can't we run into the same problem here? 

http://source.roslyn.io/#Microsoft.CodeAnalysis.VisualBasic/Binding/Binder_Conversions.vb,1074